Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -489,18 +515,35 @@ class DropDownMenu
     }
   }
 
-  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement): void
+  private static ApplyPosition (popUpDiv: HTMLDivElement, clickEvent: Nullable<MouseEvent>, referenceElement: HTMLElement, isPeriodic: boolean): void
```

Comment:
I think we should consider a different name here. From the perspective of `ApplyPosition`, it's not relevant if the call happens from a periodic operation or not. Instead, the relevant information is which kind of behavior we want - 'forced update' or 'conditional update' (we need a good name for the behavior/concept).

Additionally: We could also introduce two methods that are used in the code base and then delegate to this method with the (appropriately named) boolean parameter to make the callsites more expressive. Of course, that's not feasible for more complex overload-scenarios, but here, right now, it would work.